### PR TITLE
DOC-425 part 3: Drafts for CaVEMan, CellRanger_10x_Chromium, CellRanger_10x_Filtered_Counts, CellRanger_10x_Raw_Counts, DNAcopy workflows

### PR DESCRIPTION
**Pull request by ebusygina**
Drafts for CaVEMan, CellRanger_10x_Chromium, CellRanger_10x_Filtered_Counts, CellRanger_10x_Raw_Counts, DNAcopy workflows

### CODE REVIEW

File: docs/Encyclopedia/pages/CellRanger_10x_Raw_Counts.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# CellRanger - 10x Raw Counts
+
+## Description ##
+
+CellRanger - 10x Raw Counts is a gene expression pipeline used in GDC single-cell RNA-Seq (scRNA-Seq) harmonization.
+
+## Overview ##
+
+CellRanger - 10x Raw Counts is part of the scRNA-Seq Pipeline (single-nuclei) used for scRNA-Seq harmonization at GDC. Alignment is performed with CellRanger - 10x Raw Counts using tumor tissues and generates transcriptome profiling data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/CellRanger_10x_Raw_Counts.md:9]:
```suggestion
CellRanger - 10x Raw Counts is part of the scRNA-Seq Pipeline used for scRNA-Seq harmonization at GDC. Gene expression quantification, for all barcodes, is performed with CellRanger and generates transcriptome profiling data.
```

File: docs/Encyclopedia/pages/CellRanger_10x_Filtered_Counts.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+CellRanger - 10x Filtered Counts is a gene expression pipeline used in GDC single-cell RNA-Seq (scRNA-Seq) harmonization.
+
+## Overview ##
+
+CellRanger - 10x Filtered Counts is part of the scRNA-Seq Pipeline (single-nuclei) used for scRNA-Seq harmonization at GDC. Alignment is performed with CellRanger - 10x Filtered Counts using tumor tissues and generates transcriptome profiling data.
+
+### Input
+
+* Tumor BAM - scRNA-Seq
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/CellRanger_10x_Filtered_Counts.md:13]:
```suggestion
* BAM - scRNA-Seq
```

File: docs/Encyclopedia/pages/CellRanger_10x_Raw_Counts.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+### Input
+
+* Tumor BAM - scRNA-Seq
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/CellRanger_10x_Raw_Counts.md:13]:
```suggestion
* BAM - scRNA-Seq 
```

File: docs/Encyclopedia/pages/CellRanger_10x_Chromium.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+CellRanger - 10x Chromium is a gene expression pipeline used in GDC single-cell RNA-Seq (scRNA-Seq) harmonization.
+
+## Overview ##
+
+CellRanger - 10x Chromium is part of the scRNA-Seq Pipeline (single-nuclei) used for scRNA-Seq harmonization at GDC. Alignment is performed with CellRanger - 10x Chromium using tumor tissues and generates sequencing reads data.
+
+### Input
+
+Submitted sequencing reads
```
[the remark this answers is not in the document]
Comment (ebusygina) [reply docs/Encyclopedia/pages/CellRanger_10x_Chromium.md]: added ✅

File: docs/Encyclopedia/pages/CellRanger_10x_Filtered_Counts.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# CellRanger - 10x Filtered Counts
+
+## Description ##
+
+CellRanger - 10x Filtered Counts is a gene expression pipeline used in GDC single-cell RNA-Seq (scRNA-Seq) harmonization.
+
+## Overview ##
+
+CellRanger - 10x Filtered Counts is part of the scRNA-Seq Pipeline (single-nuclei) used for scRNA-Seq harmonization at GDC. Alignment is performed with CellRanger - 10x Filtered Counts using tumor tissues and generates transcriptome profiling data.
```
[the remark this answers is not in the document]
Comment (ebusygina) [reply docs/Encyclopedia/pages/CellRanger_10x_Filtered_Counts.md]: changed

File: docs/Encyclopedia/pages/CellRanger_10x_Raw_Counts.md (hunk 1 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/CellRanger_10x_Raw_Counts.md]: changed

File: docs/Encyclopedia/pages/CellRanger_10x_Filtered_Counts.md (hunk 2 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/CellRanger_10x_Filtered_Counts.md]: changed

File: docs/Encyclopedia/pages/CellRanger_10x_Raw_Counts.md (hunk 3 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/CellRanger_10x_Raw_Counts.md]: changed

File: docs/Encyclopedia/pages/DNAcopy.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# DNAcopy
+
+## Description ##
+
+DNAcopy is a copy number variation (CNV) analysis pipeline used in GDC genotyping array harmonization.
+
+## Overview ##
+
+DNAcopy is one of the three copy number variation analysis pipelines at the GDC. CNV analysis is performed with DNAcopy using tumor and normal alignments and generates copy number variation data.
```
[the remark this answers is not in the document]
Comment (ebusygina) [reply docs/Encyclopedia/pages/DNAcopy.md]: changed

File: docs/Encyclopedia/pages/DNAcopy.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+
+### Input
+
+Submitted sequencing reads
```
[the remark this answers is not in the document]
Comment (ebusygina) [reply docs/Encyclopedia/pages/DNAcopy.md]: changed

**Review by wwysoc2 — APPROVED** [review]
lgtm 👍

**Pull request merged**